src/main/java/org/spongepowered/common/interfaces/IMixinWorld.java

```diff
@@ -100,4 +103,12 @@
 
     void notifyBlockOfStateChange(BlockPos notifyPos, Block sourceBlock, BlockPos sourcePos);
 
+    @Nullable
+    EntityPlayer getClosestPlayerToEntityWhoAffectsSpawning(net.minecraft.entity.Entity entity, double distance);
+
+    @Nullable
+    EntityPlayer getClosestPlayerWhoAffectsSpawning(double x, double y, double z, double distance);
+
+    boolean isAnyPlayerWithinRangeAtWhoAffectsSpawning(double x, double y, double z, double range);
```

Comment:
I'd rather stick with the current name - it's just adding two words onto the original. It's not like this is an API method.